File: rides-service/src/main/java/io/simakkoi9/ridesservice/util/RegularExpressionsConstants.java
package io.simakkoi9.ridesservice.util;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class RegularExpressionsConstants {
    public static final String ADDRESS_REGEX = "^-?\\d{1,3}(\\.\\d{1,6})?,\\s?-?\\d{1,3}(\\.\\d{1,6})?$";
}

File: driver-service/src/main/kotlin/io/simakkoi9/driverservice/util/RegularExpressionsConstants.kt
package io.simakkoi9.driverservice.util

object RegularExpressionsConstants {
    const val PHONE_REGEX = "^\\+?[1-9][0-9]{7,14}$"

    const val NAME_REGEX = "^[A-Za-zА-Яа-яЁё\\s\\-']{1,70}$"

    const val CAR_NUMBER_REGEX = "^[A-ZА-Я0-9\\s\\-]{4,10}$"
}